spec/chromium-spec.js

```diff
@@ -166,6 +166,26 @@ describe('chromium feature', function() {
       b = window.open("file://" + fixtures + "/pages/window-opener-node.html", '', 'nodeIntegration=no,show=no');
     });
 
+    it('disables node integration when it is disabled on the parent window', function(done) {
+      var b;
+      listener = function(event) {
+        assert.equal(event.data, 'undefined');
```

Comment:
👍 I updated the assert to be for a `isProcessGlobalUndefined` value that is `true`.